iothub/device/samples/convention-based-samples/readme.md

```diff
@@ -79,11 +79,11 @@ The following section provides a comparison between the older non-convention-awa
 #### Using non-convention-aware API (old):
 
 ```csharp
-// Send telemetry "serialNumber".
-string serialNumber = "SR-1234";
+// Send telemetry "temperature".
+double temperature = 70;
```

Comment:
I see you switched this to a double. If you are going to do that (which I think is a good value for temperature), I think you should change the initialized value to something that doesn't look like an int.

IOW, it should be:

```suggestion
double temperature = 70.0D;
```